src/http/modules/ngx_http_rewrite_module.c

```diff
@@ -629,7 +631,7 @@ ngx_http_rewrite_if(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
 
     if (elts != lcf->codes->elts) {
         if_code = (ngx_http_script_if_code_t *)
```

Comment:
You could do the assignment and declaration here to make the diff smaller?

Reply:
Do you mean the assignment and declaration of `if_code_offset`? `lcf->codes->elts`, or a value depending on it (`if_code_offset` in this case), must be stashed before `ngx_conf_parse()` is called.

Reply:
Oh, I see what you mean. We can use `elts` to compute `if_code_offset` instead.

Reply:
This should look better. Thanks!